src/System.Private.ServiceModel/tools/scripts/PingWCFService.ps1

```diff
@@ -0,0 +1,19 @@
+﻿param (
+    [Parameter(Mandatory=$true)]
+    [string] $Url
+)
+$request = New-Object System.Net.WebCLient
+For ($i=0; $i -le 10; $i++)
+{
+    try
+    {
+        $request.DownloadFile($Url, "temp.crl")
```

Comment:
Why are we trying to download "temp.crl" here when the purpose of this script is to ping the service to see if it's alive?